iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/TapPositionModification.java

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.modification;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.network.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Objects;
+import java.util.OptionalInt;
+
+/**
+ * Allow to modify a tap phase or ratio on two or three winding transformers.
+ * For three windings you must specify the Leg.
+ *
+ * @author Nicolas PIERRE <nicolas.pierre at artelys.com>
+ */
+public class TapPositionModification extends AbstractNetworkModification {
```

Comment:
Indeed... I only need a modification on ratio tap changers. @olperr1 do we need to introduce genericity now of in a second time?